minerl/data/data_pipeline.py

```diff
@@ -359,6 +359,7 @@ def batch_iter(self,
                    seq_len: int,
                    num_epochs: int = -1,
                    preload_buffer_size: int = 2,
+                   epoch_size: Optional[int] = None,
```

Comment:
Might be worth adding this param to the docstring (though the docstring seems so out-of-date that I wouldn't insist on it)

Reply:
Sounds like a good idea to me

Reply:
Won't document because I anticipate replacing this argument with Cody's change described in this pull request. https://github.com/shwang/minerl/pull/1#discussion_r571366975